features/webvr.yml

```
@@ -0,0 +1,84 @@
+name: WebVR
+description: Accessing virtual reality (VR) devices, including sensors and head-mounted displays.
```

A suggestion to bring us more in the style of other descriptions.

I also considered including "Not to be confused with WebXR." text but I'm not sure if we want it to come up in searches relating to WebXR. I'll let you take the one you like better, or to write a new version. 🤷 

```suggestion
description: The `navigator.getVRDisplays()` API represents virtual reality (VR) devices, including sensors and head-mounted displays.
```

```suggestion
description: The `navigator.getVRDisplays()` API represents virtual reality (VR) devices, including sensors and head-mounted displays. Not to be confused with WebXR.
```